Source/WebKit/GPUProcess/graphics/WebGPU/RemoteCommandEncoder.cpp

```diff
@@ -37,13 +38,26 @@
 #include "WebGPUObjectHeap.h"
 #include <WebCore/WebGPUCommandEncoder.h>
 
+#if PLATFORM(COCOA)
+#define MESSAGE_CHECK(assertion) do { \
+    if (UNLIKELY(!(assertion))) { \
+        if (auto connection = m_gpuConnectionToWebProcess.get()) \
+            connection->terminateWebProcess(); \
+        return; \
+    } \
+} while (0)
+#else
+#define MESSAGE_CHECK(assertion)
```

Comment:
I assume it doesn't matter for this file, but should this be  `RELEASE_ASSERT(assertion)`?

Reply:
oh that's a good idea, we should still release assert on non-Cocoa platforms